src/lib/seam/components/SupportedDeviceTable/SupportedDeviceTable.element.ts

```diff
@@ -7,7 +7,7 @@ export const name = 'seam-supported-device-table'
 export const props: ElementProps<SupportedDeviceTableProps> = {
   cannotFilter: 'boolean',
   disableFilter: 'boolean',
-  brands: 'json',
+  brands: 'array',
```

Comment:
I'm not familiar with the rest of the code base but is this all we need for it to be parsed properly? I believe we parsed brand by using `JSON.parse(brands)`. I assume this changes the flow to a new means of parsing which allows for CSV? 

If that's the case, does this break the old behavior?

Reply:
Sorry I didn't see the link to `react-to-web-components`

Reply:
Yea, I forked the lib we are using to support our requirements. I may try to PR back upstream, but for now this will keep things moving for us.